### ISSUE TITLE
sdk Player for tvOS

### ISSUE THREAD

**Issue by Sofianebenabdallah** [red5pro/red5pro-ios-app#81]
Do you go to adapt the player RTMP for tvOS ?

**Comment by beetlejesss** [comment]
I don't think this is on our roadmap currently, but if there were enough demand we might look into it. I think that you'd want RTSP, which is what is used for iOS.
